pkg/cmd/ui/ui.go

```diff
@@ -153,6 +153,24 @@ func EnterSealedSecretService(sealedSecretService *types.NamespacedName) string
 	return strings.TrimSpace(sealedNs)
 }
 
+// SelectInsecureSecrets, prompts the UI to ask to generate unsealed secrets or not
+func SelectInsecureSecrets(err error) bool {
+	var insecure, msg string
+	if err != nil {
+		msg = "Do you want to use 1) unsealed secrets or 2) sealed secrets and provide the details of the Sealed Secrets Operator installation?"
+	} else {
```

Comment:
@keithchong Do we need this ? I was just thinking if the user has already installed sealed secrets in his cluster, Why would he/she opt to not use it ?

Reply:
Hi @iam-veeramalla , in my original change for this, I did not include this.  If the dependency check succeeds, then the user should just go ahead and seal the secrets.  However, the issue here is that we need the option in interactive mode for all cases.  So, for the sake of consistency, you will still get prompted.  Another reason is that, say User A set up the sealed secrets operator and controller so that dependency check succeeded. However, User B running this command is intent on using unsealed secrets.  This provides the way to do it without using the --insecure flag on the command line.  Have a look at the Jira for the detailed scenarios.

Reply:
makes sense :) , Thanks @keithchong